flixel/text/FlxText.hx

```diff
@@ -1249,4 +1249,4 @@ enum abstract FlxTextAlign(String) from String
 	}
 }
 
-private typedef AlignType = #if openfl_legacy String #else TextFormatAlign #end;
+private typedef AlignType = TextFormatAlign;
```

Comment:
We should remove this type and use TextFormatAlign, since most functions use TextFormatAlign, already

Reply:
...unless you see any reason for it to exist

Reply:
> We should remove this type and use TextFormatAlign, since most functions use TextFormatAlign, already

Ok